### ISSUE TITLE
Does epubcheck validate links

### ISSUE THREAD

**Issue by neihcnim** [w3c/epubcheck#1349]
I am a newbie to epub format and have a question regarding epubcheck.
Does epubcheck validate links (in outline) and make sure it is not  a broken link?

This is not regarding web hyperlinks but links within the epub. Thank you.

**Comment by rdeltour** [comment]
Yes, EPUBCheck does check that reative links resolve to a container resource and existing ID.

Please note: this issue tracker should only be used to report bugs or feature requests, and not for user support questions. You can find more on our website about how to [get help](https://www.w3.org/publishing/epubcheck/docs/support/) with EPUBCheck.
